spec/button_manager_spec.rb

```diff
@@ -16,7 +16,7 @@
   end
 
   it "Validate ipn message" do
-    @api.ipn_valid?("Invalid").should be_false
```

Comment:
This is needed?

Reply:
Yes! The tests where failing otherwise. (unmaintained tests, I guess.)